examples/simple-page/bemhtml-templates.js

```diff
@@ -0,0 +1,153 @@
+module.exports = function() {
+  // 1. Common HTML elements
+  // — doctype, html, head, meta, title, body
+  block('page').wrap()(function() {
+    var ctx = this.ctx;
+    var data = ctx.data;
+
+    return [
+      '<!DOCTYPE html>',
+      {
+        tag: 'html',
+        attrs: { lang: 'en-US' },
+        content: [
+          {
+            tag: 'head',
+            content: [
+              {
+                tag: 'meta',
+                attrs: { charset: 'utf-8' }
+              },
+              {
+                tag: 'title',
+                content: [
```

Comment:
Looks like you were going to do it this way with https://github.com/bem/bem-xjst/pull/229/files#diff-8c058eb20f03bea79a21ff7b4165893bR9